content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: One Year of Excalidraw
+date: 2021-01-01
+author: Excalidraw Team
+link: https://github.com/orgs/excalidraw/people
+image: og.png
+---
+
+> It's been kind of a different year, but it was the first year and pretty amazing for [Excalidraw](https://excalidraw.com).
+
+<!-- end -->
+
+Excalidraw started as a way to procrastinate on January 1st, 2020, and ended up being a fully fledged whiteboard product only one year later! In this post, we'll go over the most important features that made Excalidraw great at being a virtual whiteboard for sketching hand-drawn like diagrams.
+
+We are so incredibly proud to have built something that is being used by **20k weekly active people**.
+
+https://excalidraw.com/#json=6443031091740672,amxJZJxlZAlUBLADWIukFg
+
+## Tech Stack
+
+Excalidraw is fully open source, but it also stands on the shoulders of many other projects. We couldn't do it without [Rough.js](https://github.com/rough-stuff/rough), the library that gives Excalidraw its unique look, FG_Virgil the hand-written font designed by [Ellinor Rapp](https://www.myfonts.com/newsletters/cc/200712.html) and [React](https://github.com/facebook/react), of course. [TypeScript](https://github.com/microsoft/TypeScript) is also essential to tame the complexity of our codebase, giving a helping hand to new contributors (and old ones alike). Our full list of [dependencies](https://github.com/excalidraw/excalidraw/network/dependencies) is listed on GitHub.
+
+In addition, Excalidraw was made possible thanks to awesome services such as [Vercel](https://vercel.com/) for hosting and pull request previews, [Crowdin](https://crowdin.com/project/excalidraw) for managing dozens of translations, [CodeSandbox](https://codesandbox.io/) for easy hacking on the project, [Sentry](https://sentry.io/) for error reporting, and [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) to keep our dependencies up to date.
```

Comment:
```suggestion
In addition, Excalidraw is depending on many awesome services such as [Vercel](https://vercel.com/) for hosting and pull request previews, [Crowdin](https://crowdin.com/project/excalidraw) for managing dozens of translations, [CodeSandbox](https://codesandbox.io/) for easy hacking on the project, [Sentry](https://sentry.io/) for error reporting, and [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) to keep our dependencies up to date.
```

We have too many "couldn't be possible"/"was possible"/"couldn't do it" in these 3 paragraphs.